### DISCUSSION TITLE
How to speed up the rendering process ?

### DISCUSSION THREAD

**Discussion by Sunshine19980919** [facebookresearch/pytorch3d#922] [General]
Do you have any attempts or modifications to speed up mesh rendering? For example, delete redundant modules or judgment logic?

**Comment by bottler** [comment]
I suggest giving lots of details about what you are trying to do - how big is the mesh, image size, what type of textures and shader, what type of GPU if any, how long it currently takes, and how rendering fits in to your broader program.

**Reply by Sunshine19980919** [reply]
I'm trying to speed up rendering my mesh file, which is of the same type as the routine （https://pytorch3d.org/tutorials/render_textured_meshes）. The grid file contains three files: Obj, PNG and MTL. The size of grid file is not always determined, but it is generally less than 1m. I specified the size of the rendered image as (2048，2048). At present, GPU (1050ti) is used, and the rendering time is about 6S. At present, I don't know much about shaders, so I don't have specific requirements. For the time being, I use:

![image](https://user-images.githubusercontent.com/81543547/140954582-12a6d4e8-3850-4454-83d6-451b9b90130a.png)
